app/views/schools/add_participants/name.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title, "Enter full name to confirm" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %> - <%= @cohort.display_name %> cohort</span>
+
+    <% if add_participant_form.type == :ect %>
+      <h1 class="govuk-heading-xl">What’s the full name of this ECT?</h1>
+    <% else %>
+      <h1 class="govuk-heading-xl">What’s the full name of this mentor?</h1>
+    <% end %>
+
+      <%= form_for add_participant_form, url: { action: :update }, method: :patch do |f| %>
+        <%= f.govuk_text_field :full_name, width: "three-quarters", label: { hidden: true } %>
```

Comment:
Ditto on the hidden

Reply:
So here, the generated HTML is 
```HTML
<label for="schools-add-participant-form-full-name-field" class="govuk-label">
    <span class="govuk-visually-hidden">Full_name</span>
</label>
```
which is unlikely to read correctly on a screen reader

Reply:
Hi @ethanmills, let me know if we need to grab Brian for a quick chat about this or if you are happy with this as it is

Reply:
Follow up tickets created https://dfedigital.atlassian.net/browse/CPDRP-1022 && https://dfedigital.atlassian.net/browse/CPDRP-1023